app/views/users/show.html.erb

```diff
@@ -1,5 +1,15 @@
 <section class="dashboard-main">
   <h1> <%= current_user.first_name %>'s Dashboard </h1>
+  <% if current_user != nil %>
```

Comment:
Add conditional to show certain things based on if you have confirmed email or not